### PR TITLE
Usecase wheat production

### PR DESCRIPTION
**Pull request by geek-yang**
Replace dummy usecase with usecase wheat production.

- grab content from slides and adapt it to our web application
- fix some styling issues

### CODE REVIEW

File: content/usecaseItems.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 example_usecase_id: 0
-example_usecase_name: Example usecase
-example_usecase_description: This is an example usecase.
+example_usecase_name: Multi-year prediction of drought and heat stress to support decision making in the wheat sector
```
Comment (SarahAlidoost) [inline content/usecaseItems.md]: can we summarize the name? something like: `Multi-year prediction of drought and heat stress in the wheat sector`

File: content/usecaseItems.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+example_usecase_description: Wheat sector is heavily influenced by recent alterations in the frequency and severity of extreme climate events. There is a growing need of climate information for effective planning and adaptive actions to the near-term climate variability and change.
```
Comment (SarahAlidoost) [inline content/usecaseItems.md]: I think that now there are too much text on the first page of the use case. Can we summarize the description like: 
`The wheat sector is heavily influenced by extreme climate events. Therefore, climate information is needed to support decision-making in the wheat sector.`
Or perhaps we can replace this description with author's names and affiliation? something like `B. Solaraju-Murali, et al. Barcelona Supercomputing Center`

**Review by SarahAlidoost — APPROVED** [review]
@geek-yang nice work. I left some suggestions about the name and description of the usecase. 

The only major issue is about slides 6 and 7 which are now two separate panels.  I also noticed that the last slide **Conclusion** does not have a figure. So we should think about changing the layout of the panels to support these two cases:
- multiple images in one panel
- no image in one panel. 

What do you think? This issue can be solved in a separate PR too.

**Comment by geek-yang** [comment]
> @geek-yang nice work. I left some suggestions about the name and description of the usecase.
> 
> The only major issue is about slides 6 and 7 which are now two separate panels. I also noticed that the last slide **Conclusion** does not have a figure. So we should think about changing the layout of the panels to support these two cases:
> 
> * multiple images in one panel
> * no image in one panel.
> 
> What do you think? This issue can be solved in a separate PR too.

Thanks for your comment. Peter and I discuss the solution and we decide to create a separate conclusion page with the figure as the title page from the slides. We plan to keep it uniform for all the incoming usecases.

File: content/usecaseItems.md (hunk 1 above)
Comment (geek-yang) [reply content/usecaseItems.md]: Fixed.

File: content/usecaseItems.md (hunk 2 above)
Comment (geek-yang) [reply content/usecaseItems.md]: Good idea! We use the author names and affiliations now to replace the description!

**Review by SarahAlidoost — APPROVED** [review]
@geek-yang looks much better now, many thanks. :+1:

**Pull request merged**